lnbits/core/models/wallets.py

```diff
@@ -34,6 +34,8 @@ class Wallet(BaseModel):
     updated_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
     currency: Optional[str] = None
     balance_msat: int = Field(default=0, no_database=True)
+    icon: str = "flash_on"
```

Comment:
should go into `extra`, see: https://github.com/lnbits/lnbits/pull/2917/files#r1933635201

Reply:
Icon could one day be an image blob, so not sure a json makes sense